baize/responses.py

```diff
@@ -178,16 +178,19 @@ def parse_range(
             raise MalformedRangeHeader("Only support bytes range")
 
         ranges = [
-            (int(_[0]), int(_[1]) + 1 if _[1] else max_size)
-            for _ in re.findall(r"(\d+)-(\d*)", ranges_str)
+            (
+                int(_[0]) if _[0] else max_size - int(_[1]),
```

Comment:
🤔 Why use `max_size - int(_[1])`?

Reply:
This is to handle the suffix case. 
`Range: bytes=-10` should return the last 10 bytes of the file. The start will be the suffix distance from the end of the file, which the regex captures as _[1].